sipa/model/wu/ldap_utils.py

```diff
@@ -128,10 +128,15 @@ def change_email(username, password, email):
     attribute, if it is not yet set up (replace removes all
     attributes of the given kind and puts the new one in place)
     """
+    delete = email is None
     try:
         with LdapConnector(username, password) as l:
-            l.modify(dn=l.get_dn(),
-                     changes={'mail': [(ldap3.MODIFY_REPLACE, [email])]})
+            if delete:
+                l.modify(dn=l.get_dn(),
+                         changes={'mail': [(ldap3.MODIFY_DELETE, [])]})
+            else:
+                l.modify(dn=l.get_dn(),
+                         changes={'mail': [(ldap3.MODIFY_REPLACE, [email])]})
```

Comment:
Slightly shorter (and focusing the `try`-Block more towards the ldap calls) would be
```python
mail_change = (ldap3.MODIFY_DELETE, []) if delete else (ldap3.MODIFY_REPLACE, [email])
try:
    with LdapConnector(username, password) as l:
        l.modify(dn=l.get_dn(), changes={'mail': [mail_change]})
```

Also, please add a note to the docstring that `mail is None` now changes behavior from `MODIFY_REPLACE` to `MODIFY_DELETE`.